PVPCPlanner/View/Screen/SettingsView.swift

```diff
@@ -3,6 +3,7 @@ import SwiftUI
 struct SettingsView: View {
     @Environment(\.colorScheme) private var colorScheme
     @AppStorage(AppStorageKeys.THEME_MODE.rawValue) var selectedTheme: ThemeMode = .auto
+    @AppStorage(AppStorageKeys.LOCATION.rawValue) var selectedLocation: Locations = .MainlandAndIslands
```

Comment:
Esto es lo que habéis estado hablando no? no se puede sacar al viewmodel? para que sea como una consulta aparte y tal?

Reply:
Solo hay que llevarlo al viewmodel

Reply:
A ver, se puede dejar aquí la variable y que los condicionales lo haga desde el vm, porque tampoco hay mucha mas logica que vaya sin la magia de los decoradores, pero que diga Oli que esto lo maneja mas

Reply:
@Popitus manifiéstate!

Reply:
los AppStorage se usan en la vista, pero no tiene lógica alguna. solo almacena un valor primitivo. Todo lo que sea trabajar con ello es dentro del vm. Eso es lo que yo siempre he hecho. Que se puede hacer en vm.. si. pero feo es ....

Reply:
Bueno, pues adelante con la fealdad de iOS.